test/com/google/javascript/jscomp/CommandLineRunnerTest.java

```diff
@@ -1406,37 +1405,37 @@ public void testProcessCJSWithModuleOutput() {
    */
   public void testProcessCJSWithClosureRequires() {
     args.add("--process_common_js_modules");
-    args.add("--common_js_entry_module=app.js");
-    args.add("--manage_closure_dependencies");
+    args.add("--entry_point=app");
+    args.add("--dependency_mode=STRICT");
     setFilename(0, "base.js");
     setFilename(1, "array.js");
     setFilename(2, "Baz.js");
     setFilename(3, "app.js");
     test(new String[] {
-           "/** @provideGoog */\n" +
-           "/** @const */ var goog = goog || {};\n" +
-           "var COMPILED = false;\n" +
-           "goog.provide = function (arg) {};\n" +
-           "goog.require = function (arg) {};\n",
-
-           "goog.provide('goog.array');\n",
-
-           "goog.require('goog.array');\n" +
-           "function Baz() {}\n" +
-           "Baz.prototype = {\n" +
-           "  baz: function() {\n" +
-           "    return goog.array.last(['asdf','asd','baz']);\n" +
-           "  },\n" +
-           "  bar: function () {\n" +
-           "    return 4 + 4;\n" +
-           "  }\n" +
-           "}\n" +
-           "module.exports = Baz;",
-
-           "var Baz = require('./Baz');\n" +
-           "var baz = new Baz();\n" +
-           "console.log(baz.baz());\n" +
-           "console.log(baz.bar());\n"
+           joinStrings("/** @provideGoog */",
```

Comment:
I try to avoid making code style changes in the same PR as a feature change - especially in tests. We have a lot of tests which don't quite meat current style guides.